### DISCUSSION TITLE
Excessive views on repo

### DISCUSSION THREAD

**Discussion by zhannaavanesova** [community/community#130651] [Other Feature Feedback, Questions, & Ideas]
Hi all, over the last 2 months I've noticed 2 of my public repos have consistently (every single day) had 470+ views with only one or two unique viewers (seen under Insights -> Traffic). This number of views is uncommon for these repos, and I think its a little strange its all from one unique viewer and a similar number every day. Seems like something automated. Has anyone else experienced something like this? Wondering if this is a security concern/ideas on where to look. Thanks in advance.

**Comment by Aryan-Falcon** [comment]
When I checked my views too, I saw there were 54 views with 1 unique viewer. I think that the views count how much times that the repository is visited, even for testing or if your visiting your own repo, while the number of unique viewers is the amount of actual different IP addresses who have visited it. This is just a suggestion, because even my 54 views seems a little too much.

**Comment by catunderrain** [comment]
Yeah, I see your problem. There are several possibilities. It could be automated bots crawling your repos for data, especially if your repos contain valuable information or popular code. Or, someone might be using automated tools to track your repo activity because they find vulnerabilities in your code or are monitoring changes. Alternatively, I have a visitor counter markdown code line that actually counts the times you access your repo (like reload, commit, review markdown file, etc.). As a result, one night while fixing my README.md file, I got 1000+ views (actually the times visited). So, if you see that there's nothing too serious that needs to be secured, don't worry that someone is trying to do something bad to you bro lol
![image](https://github.com/user-attachments/assets/23e1ed59-af62-450f-b197-0b83de4c8f95)
